Code/Tools/AssetProcessor/native/ui/ProductAssetDetailsPanel.h

```diff
@@ -57,6 +59,20 @@ namespace AssetProcessor
         }
 
         void SetScanQueueEnabled(bool enabled);
+        void SetupDependencyGraph(QTreeView* productAssetsTreeView, AZStd::shared_ptr<AssetDatabaseConnection> assetDatabaseConnection);
+
+        ProductDependencyTreeModel* GetOutgoingDependencyTreeModel() const
+        {
+            return m_outgoingDependencyTreeModel;
+        }
+
+        ProductDependencyTreeModel* GetIncomingDependencyTreeModel() const
+        {
+            return m_incomingDependencyTreeModel;
+        }
```

Comment:
probably better to use a helper method or forward a signal of some kind so you don't expose the underlying ui elements.

Reply:
I did not notice all uses of these methods were removed. I have removed these methods. 
Thanks for pointing it out and I will keep in mind not to expose the entire element in the future.